### DISCUSSION TITLE
Figma desktop app stopped working

### DISCUSSION THREAD

**Discussion by UnicornSeal** [Moonvy/figmaEX#6] [Q&A] [answered]
Hello! 

@staff-moonvy, thank you for developing such a unique and convenient extension for Figma!
I just have a little trouble now - after installing it, my Figma app stopped working and I can't run it anymore, even after reinstalling it...
Have you experienced something like that before?

![Без імені](https://user-images.githubusercontent.com/82653042/115017260-ef423300-9ebe-11eb-9201-0c50cd2fd903.png)

**Comment by justjegor** [comment]
I have the same problem!!!

**Reply by NikolayGur** [reply]
Good afternoon. You need to rename the figma folder: %UserProfile%/AppData/Local/Figma to any name. And after that you can install figma again.

**Reply by Reza00498895** [reply]
Is this method working?

**Reply by NewChristmas** [reply]
Works

**Reply by haroldao** [reply]
Ty! It works!

**Comment by yArna** [comment]
- What is your Figma version?
- You can click on "Open Setup Log" to provide a log so that we can know the details.

- If you want to fix your Figma, You can try using the `2. Restore Backup` option in the FIgmaEX installer

```node
....
select command （选择一个操作） (Use arrow keys)
  1. Start Install（安装） 
❯ 2. Restore Backup（还原备份） 
```

- If that still doesn't work, you can delete the Figma installation directory `%UserProfile%/AppData/Local/Figma`  and reinstall Figma, which should allow you to completely reinstall Figma

**Comment by yArna** [comment]
We updated the installer [FigmaEX v1.0.3](https://github.com/Moonvy/figmaEX/releases/tag/v1.0.3) , should be able to uninstall normally with him

**Comment by rraggio90** [comment]
Figma Ex for desktop stop working, I restored, unsitalled both the Figma Ex and Figma desktop application and if figma Ex is installed Figma wont open, when I select restore Figma I can open my figma but the side bar is not there....
I have the lasted version of figma 101.6

**Reply by yArna** [reply]
Because of the restrictions added to Figma Desktop App v101.6 and above
Cannot be installed on Figma Desktop App v101.6 or above 
If you need to use it, please [download the lower version of Figma Desktop App](https://github.com/Moonvy/figmaEX/blob/master/figma-desktop-help.md).

Download Figma 99 below version:

- https://github.com/kogoro-mouri/Figma-Download-Mirror
- https://gitee.com/kogoro/figma-download-mirror
- https://figma.software.informer.com/download/
- https://www.filehorse.com/download-figma/


Run FigmaEX installer without running Figma

For Figma features, the low version client is the same as the latest version client

> on Windows Put FigmaEX Installer in the same folder as Figma

**Comment by rraggio90** [comment]
Could you send me the link to download Figma previous version for a mac?
when will you update Figma Ex to run with the sidebar

On Fri, Aug 20, 2021 at 2:22 PM yArna ***@***.***> wrote:

> Because of the restrictions added to Figma Desktop App v101.6 and above
> Cannot be installed on Figma Desktop App v101.6 or above
> If you need to use it, please download the lower version of Figma Desktop
> App <https://github.com/Moonvy/figmaEX/blob/master/figma-desktop-help.md>.
>
> Download Figma 99 below version:
>
>    - https://gitee.com/kogoro/figma-download-mirror
>    - https://github.com/kogoro-mouri/Figma-Download-Mirror
>    - https://figma.software.informer.com/download/
>    - https://www.filehorse.com/download-figma/
>
> Run FigmaEX installer without running Figma
>
> on Windows Put FigmaEX Installer in the same folder as Figma
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Moonvy/figmaEX/discussions/6#discussioncomment-1213611>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASZLG3P5E22OCYNOEX52LELT52FMXANCNFSM43BK7SQQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>
> The information contained in this electronic message and any attachments
> to this message are intended for the exclusive use of the addressee(s) and
> may contain proprietary, confidential or privileged information. If you are
> not the intended recipient, you should not disseminate, distribute or copy
> this e-mail. Please notify the sender immediately and destroy all copies of
> this message and any attachments. Any views or opinions contained in this
> message are those of the author only and do not necessarily represent the
> views or policies of the company.
>
>
>
>
> WARNING: Computer viruses can be transmitted via email. The recipient
> should take responsibility to check this email and any attachments for the
> presence of any viruses before they accept into their network / systems.
>

-- 


---------------Disclaimer---------------



The information contained in 
this electronic message
and any attachments to this message are intended 
for the exclusive use of the
addressee(s) and may contain proprietary, 
confidential or privileged
information. If you are not the intended 
recipient, you should not disseminate,
distribute or copy this e-mail. 
Please notify the sender immediately and
destroy all copies of this message 
and any attachments. Any views or opinions
contained in this message are 
those of the author only and do not necessarily
represent the views or 
policies of the company.





WARNING: Computer viruses can be transmitted 
via
email. The recipient should take responsibility to check this email and 
any
attachments for the presence of any viruses before they accept into 
their network / systems.



-----------------------------------------

**Reply by yArna** [reply]
Sorry, this is an official Figma restriction, I tried to release an integration with the latest version of the Figma client, but this was not in line with Figma policy and I removed it as requested by Figma staff  #21

**Comment by yArna** [comment]
@rraggio90 

Here's the link you need

- [Figma-99.0.0.MacOS.zip](https://github.com/kogoro-mouri/Figma-Download-Mirror/raw/master/dist/MacOS/Figma-99.0.0.MacOS.zip)

Please use FigmaEX-Installer-.macOS.-v1.2.5-b 
 
- [FigmaEX-Installer-.MacOS.-v1.2.5-b](https://github.com/Moonvy/figmaEX/releases/download/v1.2.5/FigmaEX-Installer-.MacOS.-v1.2.5-b.zip)

**Comment by rraggio90** [comment]
IT IS STILL NOT WORKING

On Fri, Aug 20, 2021 at 2:31 PM yArna ***@***.***> wrote:

> @rraggio90 <https://github.com/rraggio90>
>
> Here's the link you need
>
>    - Figma-99.0.0.MacOS.zip
>    <https://github.com/kogoro-mouri/Figma-Download-Mirror/raw/master/dist/MacOS/Figma-99.0.0.MacOS.zip>
>
> Please use FigmaEX-Installer-.macOS.-v1.2.5-b
>
>    - FigmaEX-Installer-.MacOS.-v1.2.5-b
>    <https://github.com/Moonvy/figmaEX/releases/download/v1.2.5/FigmaEX-Installer-.MacOS.-v1.2.5-b.zip>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Moonvy/figmaEX/discussions/6#discussioncomment-1213637>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASZLG3LRKZVW6XCHEHR5KR3T52GO5ANCNFSM43BK7SQQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>
> The information contained in this electronic message and any attachments
> to this message are intended for the exclusive use of the addressee(s) and
> may contain proprietary, confidential or privileged information. If you are
> not the intended recipient, you should not disseminate, distribute or copy
> this e-mail. Please notify the sender immediately and destroy all copies of
> this message and any attachments. Any views or opinions contained in this
> message are those of the author only and do not necessarily represent the
> views or policies of the company.
>
>
>
>
> WARNING: Computer viruses can be transmitted via email. The recipient
> should take responsibility to check this email and any attachments for the
> presence of any viruses before they accept into their network / systems.
>

-- 


---------------Disclaimer---------------



The information contained in 
this electronic message
and any attachments to this message are intended 
for the exclusive use of the
addressee(s) and may contain proprietary, 
confidential or privileged
information. If you are not the intended 
recipient, you should not disseminate,
distribute or copy this e-mail. 
Please notify the sender immediately and
destroy all copies of this message 
and any attachments. Any views or opinions
contained in this message are 
those of the author only and do not necessarily
represent the views or 
policies of the company.





WARNING: Computer viruses can be transmitted 
via
email. The recipient should take responsibility to check this email and 
any
attachments for the presence of any viruses before they accept into 
their network / systems.



-----------------------------------------

**Reply by yArna** [reply]
Make sure you have downloaded version 99 of Figma（ [Figma-99.0.0.MacOS.zip](https://github.com/kogoro-mouri/Figma-Download-Mirror/raw/master/dist/MacOS/Figma-99.0.0.MacOS.zip)）

Make sure you are using the latest Installer（ [FigmaEX-Installer-.MacOS.-v1.2.5-b](https://github.com/Moonvy/figmaEX/releases/download/v1.2.5/FigmaEX-Installer-.MacOS.-v1.2.5-b.zip)
）

**Reply by lbsept** [reply]
亲亲 我陷入了一个死循环，下载了99 也安装了最新的EX。 但是哈，启动figma的时候会问现在立刻更新或者下次打开更新。不管怎么选，下次在开启都是自动就升级了😂 求大神支招

**Reply by yArna** [reply]
你是否下载了 FigmaEX-Installer-.MacOS.-v1.2.5-b，你安装 99 版本后不要启动，紧接着使用 FigmaEX-Installer-.MacOS.-v1.2.5-b

**Reply by lbsept** [reply]
按照这个步骤，人间完美。现在又可以用了感谢感谢感谢

**Comment by yArna** [comment, marked as answer]
# How to Install for Figma Desktop

Because of the restrictions added to Figma Desktop App v101.6 and above Cannot be installed on Figma Desktop App v101.6 or above If you need to use it, please download the lower version of Figma Desktop App.

1. Download Figma 99 below version:

- https://github.com/kogoro-mouri/Figma-Download-Mirror
- https://gitee.com/kogoro/figma-download-mirror
- https://figma.software.informer.com/download/
- https://www.filehorse.com/download-figma/


2. Don't running Figma Desktop App (avoid it auto update), First Run [FigmaEX installer1.25-b](https://github.com/Moonvy/figmaEX/releases/tag/v1.2.5).

> on Windows Put FigmaEX Installer in the same folder as Figma Desktop App.

For Figma features, the low version client is the same as the latest version client

**Reply by Flashwalker** [reply]
Could you explain more verbose please:  
-Which variant of  **FigmaEX** installer ( cli or gui or no matter ) should i put in the same folder as **Figma**?  
-And should i put it exact in the folder with **Figma.exe** binary?  
-And how about Figma Beta, is there a mirror for it?

Thanks.

**Comment by Flashwalker** [comment]
> Hello!
> 
> @staff-moonvy, thank you for developing such a unique and convenient extension for Figma!
> I just have a little trouble now - after installing it, my Figma app stopped working and I can't run it anymore, even after reinstalling it...
> Have you experienced something like that before?
> 
> ![Без імені](https://user-images.githubusercontent.com/82653042/115017260-ef423300-9ebe-11eb-9201-0c50cd2fd903.png)

ℹ I found that re-installation fail it is about:  
`"%USERPROFILE%\AppData\Local\Figma\app-101.6.0\resources\app.asar\_codesign"`  
and `"%USERPROFILE%\AppData\Local\Figma\app-101.6.0\resources\app.asar.unpacked\_codesign"`  
files.  
They can't be deleted causing error: `file not found`.  
I found the only way to delete them via Win fu*ing Rar. Just archive them with an option: ✅ _Delete files after archiving_  
![Screenshot Winrar-2](https://user-images.githubusercontent.com/2188215/130738326-f182b20f-50f7-414d-b07a-1afdb8a66440.PNG)
and then delete them.

**Reply by yArna** [reply]
## For Can't Install Figma
you can try  rename `%USERPROFILE%\AppData\Local\Figma` to any name for delete them.

## For Install Figma FigmaEX
- You must [download an older version(v99) of Figma](https://github.com/kogoro-mouri/Figma-Download-Mirror)
- Don't running Figma Desktop App (avoid it auto update), First Run FigmaEX installer1.25-b.
![image](https://user-images.githubusercontent.com/82231420/130748515-548e291b-0869-4ab1-94cb-af68741f8140.png)

**Comment by auau3** [comment]
EX1.25b和中文客户端版冲突，安装之后汉化和EX都失效了，之前官方99版本先汉化后装EX能共存，关键官方给的Mac汉化+EX版本不分intel和M1，希望改进。顺便问一下CLI和Cosynode版本有什么用

**Reply by yArna** [reply]
有时间我们打包一个 M1 的版本。CLI 和 Cosynode 的程序打包形式不一样，功能时一样的，为了有的人如果有不能运行的情况可以用另一种。

**Reply by auau3** [reply]
🙏感谢

**Comment by rraggio** [comment]
This does NOT WORK ON MACS, even with the older version of figma 99, please fix this

**Reply by sxchacha** [reply]
Because of the restrictions added to Figma Desktop App v101.6 and above Cannot be installed on Figma Desktop App v101.6 or above If you need to use it.
[please download the lower version of Figma Desktop App](https://github.com/Moonvy/figmaEX/blob/master/figma-desktop-help.md)

**Reply by rraggio** [reply]
I have downloaded the figma v99 and it doesnt work either!
